pages/models.py

```diff
@@ -58,13 +58,16 @@ def render(self, field, context):
         return render_str(field, context)
 
 
-class PageManager(mptt_models.TreeManager):
+class PageQuerySet(mptt.querysets.TreeQuerySet):
+    def active(self):
+        return self.filter(is_active=True)
 
-    def get_active(self):
-        return self.get_queryset().filter(is_active=True)
 
+class PageManager(models.Manager.from_queryset(PageQuerySet)):
+    pass
```

Comment:
Let's create `active` method here too to avoid excess code `Page.objects.all().active() -> Page.objects.active()`